twiggy/tests/expectations/monos_wasm_csv

```diff
@@ -0,0 +1 @@
+generic,approximate_monomorphization_bloat_bytes,approximate_monomorphization_bloat_percent,total_size,total_size_percent,monomorphizations
```

Comment:
I think this is missing the `test!(...)` for this expectation file? Also, it would be good to have a test that actually has some output.